bin/install_travis.sh

```diff
@@ -12,8 +12,13 @@ if [[ "${WITH_SAGE}" != "yes" ]]; then
     conda config --set always_yes yes --set changeps1 no;
     conda update -q conda;
     conda info -a;
-
-    conda create -q -n test-environment python="${PYTHON_VERSION}" pip cython sympy nose pytest;
+    CONDA_PKGS="pip cython sympy nose pytest"
+    if [[ "${SKIP_NUMPY}" == "yes" ]]; then
```

Comment:
I think the option should be `WITH_NUMPY` like other optional dependencies and add `WITH_NUMPY=yes` to a few existing tests.